backend/configs/settings.py

```diff
@@ -170,4 +171,6 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/2.2/howto/static-files/
 
-STATIC_URL = '/static/'
\ No newline at end of file
+STATIC_URL = '/static/'
+
+REST_FRAMEWORK = {'DEFAULT_SCHEMA_CLASS': 'rest_framework.schemas.coreapi.AutoSchema' }
```

Comment:
With this, you are rewriting the same variable on line 84.

Reply:
Another mystery. I was getting the following error without this variable. 
AttributeError: 'AutoSchema' object has no attribute 'get_link'

I checked it today and it's working without any error.   
![swagg](https://user-images.githubusercontent.com/32267051/100416399-96593600-30a8-11eb-8733-b0ea064fcaad.png)

I have commented out this variable.

Reply:
`REST_FRAMEWORK = {'DEFAULT_SCHEMA_CLASS': 'rest_framework.schemas.coreapi.AutoSchema' }`
On line 94 we already have this variable. Instead of commenting, you should add its key/value pair to it (line # 94).

Reply:
Value is added